mllib/src/main/scala/org/apache/spark/mllib/evaluation/MultilabelMetrics.scala

```diff
@@ -179,3 +167,160 @@ class MultilabelMetrics @Since("1.2.0") (predictionAndLabels: RDD[(Array[Double]
   @Since("1.2.0")
   lazy val labels: Array[Double] = tpPerClass.keys.toArray.sorted
 }
+
+
+/**
+ * Trait for statistical summary for multi-label metrics.
+ */
+private[evaluation] trait MultilabelSummary {
```

Comment:
Is there much value in splitting the trait and implementation? it's private and used in one place.